src/OpenTK/Platform/Windows/API.cs

```diff
@@ -136,6 +136,8 @@ internal static class Functions
     {
         #region Window functions
 
+        #region Drag functions
```

Comment:
We don't need more regions in the code, and we're planning to get rid of them anyway. No need to add them for individual methods. Please remove them.